### PR TITLE
feat: Posting to the /bookings endpoint

### PR DESCRIPTION
**Pull request by danjohnson95**
Fixes designmynight/dmn#24214

Documentation now includes information on storing bookings/enquiries by posting to the `/bookings` endpoint.

### CODE REVIEW

File: site/content/api/booking-api/index.md [hunk 1]
```diff
@@ -139,6 +139,83 @@ Field | Description
...
+`email` | No | `string` The guest's email address
+`dob` | No | `string` The guest's date of birth, in the format YYYY-MM-DD
+`phone` | No | `string` The guest's phone number
+`notes` | No | `string` Any additional booking notes
+
+Here's an example request of storing an enquiry for 4 people on the 16th March 2018 at 16:00
+
+```bash
+$ curl -X POST https://api.designmynight.com/v4/bookings
+    -d "source=partner&first_name=Dan&last_name=Johnson&num_people=4&type=58c927215ee246985eb91b8e&date=2018-03-16&time=16:00"
```
Comment (StevePorter92) [inline site/content/api/booking-api/index.md]: Worth adding venue id and venue group to this request?

Comment (danjohnson95) [reply site/content/api/booking-api/index.md]: Oops, missed off the `venue_id`, thanks. Do we need to put `venue_group` on there? It’s not a required field

Comment (StevePorter92) [reply site/content/api/booking-api/index.md]: Hmm good question. I had assumed it was. Does the venue group still get set if it hasn't been passed one?

Comment (danjohnson95) [reply site/content/api/booking-api/index.md]: Just checked - yes it does 🎉

**Pull request merged**